### ISSUE TITLE
projectIds on a new round are still cryptic

### ISSUE THREAD

**Issue by codenamejason** [gitcoinco/grants-stack#1323]
**Overview**
The project ids that are coming from the graph are cryptic and not readable.

**Steps To Reproduce**
```graphql
query GetPayouts($roundId: String) {
          payoutStrategies(
            where:{
              round_:{
                id: $roundId
              }
            }
          ) {
            payouts {
              id
              version
              createdAt
              token
              amount
              grantee
              projectId
              txnHash
            }
          }
        }
```

RoundId: 0x340d71C7f1B069eA43770998f209e4ff0D72c9B5
ChainId: 5

**Observed  behavior**
See screenshot

**Expected behavior**
Project id's to be in proper format

**Screenshots**
![image](https://user-images.githubusercontent.com/9419140/229169775-8cb57a29-0c1d-4e14-a25b-1a4cfc5881c7.png)

**Comment by thelostone-mc** [comment]
<img width="1295" alt="image" src="https://user-images.githubusercontent.com/5358146/229345286-e208b8c5-6cc6-4de1-83a3-7a0467113383.png">

```

{
  payoutStrategies(where:{
    id: "0x340d71c7f1b069ea43770998f209e4ff0d72c9b5"
  }) {
    payouts {
      id
      projectId
    }
  }
}
```

**Comment by thelostone-mc** [comment]
This has been fixed in commit 7b570b64adae9085b26be4809197e07d69562948
